### ISSUE TITLE
When to release the diffuers version？

### ISSUE THREAD

**Issue by ykj467422034** [Jonseed/ComfyUI-Detail-Daemon#7]
Hello, can you give a version to run with code？

**Comment by NicholasCao** [comment]
@Jonseed         sigma_float = float(sigma.max().detach().cpu())
        if end_sigma <= sigma_float <= start_sigma:
            sigma = sigma * (1.0 + lss_dishonesty_factor)

**Comment by Jonseed** [comment]
@NicholasCao I don't know. I don't know much about diffusers. Do you know @blepping ?

**Comment by blepping** [comment]
I don't know Diffusers either but it looks like that code is just changing the schedule. To get the Detail Daemon effect you'd need to sample with one set of sigmas and pass the model an adjusted value. I don't know how you'd do that with Diffuses though.

**Comment by blepping** [comment]
Actually, I looked at the Diffusers code you linked. I think it would be the first one (the model's input parameters). If you did something like:
```python
  timestep=(timestep / 1000) * (1.0 + lss_dishonesty_factor),
```
that might work. If that's actually timesteps they'd be 0 at the start of sampling and 1000 at the end. If you wanted, you could add logic to only do the adjustment when the timestep was in a certain range.

You'd have to figure out a way to plumb the extra parameters to where they're needed or just hardcode them I guess.

**Comment by NicholasCao** [comment]
> Actually, I looked at the Diffusers code you linked. I think it would be the first one (the model's input parameters). If you did something like:
> 
> ```python
>   timestep=(timestep / 1000) * (1.0 + lss_dishonesty_factor),
> ```
> 
> that might work. If that's actually timesteps they'd be 0 at the start of sampling and 1000 at the end. If you wanted, you could add logic to only do the adjustment when the timestep was in a certain range.
> 
> You'd have to figure out a way to plumb the extra parameters to where they're needed or just hardcode them I guess.

thank you！here is a solution https://github.com/huggingface/diffusers/issues/10112
